operator/v1/types_machineconfiguration.go

```diff
@@ -56,3 +64,94 @@ type MachineConfigurationList struct {
 	// Items contains the items
 	Items []MachineConfiguration `json:"items"`
 }
+
+type ManagedBootImages struct {
+	// machineManagers is an array of machineManager objects.
+	// The MCO will watch for changes to this list and enroll/un-enroll machine management resources. An entry in this list
```

Comment:
Enroll and un-enroll from? Also nit, I'm thinking maybe register and deregister, not sure I like "un-enroll"